cmd/main.go

```diff
@@ -18,7 +18,7 @@ func main() {
 		log.Fatalf("%s", err.Error())
 	}
 
-	if err := app.Run(appConfig.Port); err != nil {
-		log.Fatalf("%s", err.Error())
+	if err := app.Run(); err != nil {
+		//log.Fatalf("%s", err.Error())
```

Comment:
?

Reply:
Now app.Run() should do error logging instead of main.go. Because some app logs (for example about services stopping after errors) may happen after error logs